src/System.Management.Automation/utils/PInvokeDllNames.cs

```diff
@@ -17,6 +17,7 @@ internal static class PinvokeDllNames
         internal const string QueryDosDeviceDllName = "api-ms-win-core-file-l1-1-0.dll";                     /*1*/
         internal const string CreateSymbolicLinkDllName = "api-ms-win-core-file-l2-1-0.dll";                 /*2*/
         internal const string GetOEMCPDllName = "api-ms-win-core-localization-l1-2-0.dll";                   /*3*/
+        internal const string GetACPDllName = "api-ms-win-core-localization-l1-2-0.dll";                   /*3*/
```

Comment:
The `/*3*/` at the end is important to track if there is a mismatch. Please add this API to the end of the list and use an incremented number.